src/main/java/org/broadinstitute/hellbender/tools/genome/SparkGenomeReadCounts.java

```diff
@@ -172,7 +174,10 @@ private void collectReads(final JavaSparkContext ctx) {
 
     protected static SAMSequenceDictionary dropContigsFromSequence(final SAMSequenceDictionary originalSequenceDictionary, final Set<String> nonAutosomalContigs) {
         final List<SAMSequenceRecord> initialSequences = new ArrayList<>(originalSequenceDictionary.getSequences());
-        final List<SAMSequenceRecord> finalSequences = initialSequences.stream().filter(s -> !nonAutosomalContigs.contains(s.getSequenceName())).collect(Collectors.toList());
+        final List<SAMSequenceRecord> finalSequences = initialSequences.stream()
+                .filter(s -> !nonAutosomalContigs.contains(s.getSequenceName()))
+                .filter(s -> !StringUtils.startsWith(s.getSequenceName(), "GL") && !StringUtils.startsWith(s.getSequenceName(), "NC_"))
```

Comment:
Got in the habit of using StringUtils, for startsWith, from way back when

Done